executor/runner/runner.go

```diff
@@ -308,17 +309,15 @@ func (r *Runner) runContainer(ctx context.Context, startTime time.Time, updateCh
 }
 
 func (r *Runner) maybeSetDefaultTags() {
-	if r.dockerCfg.IsEnableDefaultTitusTags() {
-		// Initialize metrics.Reporter with default tags t.jobId and t.taskId
-		tagger, ok := r.metrics.(tagger)
-		if ok {
-			tags := map[string]string{
-				"t.jobId":  r.container.TitusInfo.TitusProvidedEnv["TITUS_JOB_ID"],
-				"t.taskId": r.container.TaskID,
-			}
-			tagger.append(tags)
-			r.logger.Infof("Set Atlas default tags to: %s", tags)
+	// If extra Atlas tags is enabled, initialize metrics.Reporter with default tags t.jobId and t.taskId
+	jobID := r.container.TitusInfo.TitusProvidedEnv["TITUS_JOB_ID"]
+	if r.metricsTagger != nil && len(jobID) > 0 {
+		tags := map[string]string{
+			"t.jobId":  jobID,
+			"t.taskId": r.container.TaskID,
 		}
+		r.metricsTagger.append(tags)
+		r.logger.Infof("Set Atlas default tags to: %s", tags)
```

Comment:
`r.logger.WithFields(tags).Info("Set Atlas default tags")`

Reply:
can't assign map[string]string to map[string]interface{}

Reply:
Ok. :+1: